src/components/TopBar.js

```diff
@@ -499,7 +513,56 @@ export default _.flow(
       }, ['Click confirm to remove button forever.']),
       openCookiesModal && h(CookiesModal, {
         onDismiss: () => this.setState({ openCookiesModal: false })
+      }),
+      openFirecloudModal && h(PreferFirecloudModal, {
+        onDismiss: () => this.setState({ openFirecloudModal: false }),
+        authState
       })
     ])
   }
 })
+
+const PreferFirecloudModal = ({ onDismiss, authState }) => {
+  const [emailAgreed, setEmailAgreed] = useState(true)
+  const [reason, setReason] = useState('')
+  const [submitting, setSubmitting] = useState(false)
+
+  const { profile: { email, firstName, lastName } } = authState
+  const returnToLegacyFC = _.flow(
+    withErrorReporting('Error opting out of Terra'),
+    Utils.withBusyState(setSubmitting)
+  )(async () => {
+    await Ajax().User.profile.preferLegacyFirecloud()
+    if (emailAgreed === true || reason.length !== 0) await Ajax().User.createSupportRequest({
+      name: `${firstName} ${lastName}`,
+      email,
+      description: reason,
+      subject: 'Opt out of Terra',
+      type: 'survey',
+      attachmentToken: '',
+      emailAgreed
+    })
+    onDismiss()
+    window.location.assign(getConfig().firecloudUrlRoot)
+  })
+
+  return h(Modal, {
+    onDismiss,
+    title: 'Take me back to classic FireCloud!',
+    okButton: returnToLegacyFC
+  }, [
+    'Are you sure you would prefer the previous FireCloud interface?',
+    h(FormLabel, ['(Optional) Please tell us why']),
```

Comment:
```suggestion
    h(FormLabel, ['Please tell us why']),
```
Since the "Ok" button is active, it is clear that this field is not required.